package.json

```diff
@@ -91,7 +92,8 @@
     "lint": "yarn lint:js && yarn lint:style && yarn lint:prettier",
     "lintfix:prettier": "prettier --write --list-different .",
     "lintfix": "prettier --write --list-different . && yarn lint:js --fix && yarn lint:style --fix",
-    "prepare": "husky install",
-    "test": "jest"
+    "prepare": "husky install && yarn sync",
+    "test": "jest",
+    "sync": "node -r dotenv/config ./build-utils/sync-storefront.mjs"
```

Comment:
Shouldn't `sync` run before `build` and/or `generate` each time to ensure configs are up-to-date?